.github/workflows/docs.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches: 
     - master
+    - docs_update
```

Comment:
hmm... don't _love_ the idea of reusing the branch name over and over. but serves the purpose for your example

Reply:
I know. But is the best way to protect on main repo and execute the workflow. Open to suggestions.